src/SFA.DAS.ApplyService.Web/Views/Shared/_ValidationSummaryPartial.cshtml

```diff
@@ -1,4 +1,4 @@
-﻿@if (!ViewData.ModelState.IsValid)
+﻿@if (!ViewData.ModelState.IsValid && ViewData.ModelState.Any(e=>e.Value.Errors.Count > 0))
```

Comment:
Why did we need this additional check ?

Reply:
There are few scenarios where modelstate is just invalid without any errors, so page is displaying 'there is a problem' error, to avoid that this condition is added